CMakeLists.txt

```
@@ -406,6 +406,27 @@ if(ENABLE_TESTS OR ENABLE_BENCHMARKS)
     target_include_directories(hacl_cpu_features PUBLIC ${PROJECT_SOURCE_DIR}/cpu-features/include)
 endif(ENABLE_TESTS OR ENABLE_BENCHMARKS)
 
+# Add ecckiila for benchmarks
+if(ENABLE_TESTS OR ENABLE_BENCHMARKS)
```

You probably don't want these for tests but only benchmarks.
Same for the others below.

```suggestion
if(ENABLE_BENCHMARKS)
```